### ISSUE TITLE
Minimize to system tray

### ISSUE THREAD

**Issue by bmix** [fman-community/fman#144]
I would like to hide _fman_ to the right part of the taskbar, where all the tiny little icons are (don't know the name, right now) and make it come up wherever I want with a global keyboard shortcut. Would be similar to 'Quake style' terminals.

**Comment by mherrmann** [comment]
I had to google it myself but it's called the notification area I think. You're talking about Windows, correct?

What exactly do you mean by _wherever_ in "make it come up wherever I want"? Do you mean that fman should display the contents of _wherever_? Or do you want to be able to open fman independently of where you currently are (eg. which app you have open) in the OS?

**Comment by bmix** [comment]
Yep, notification area on Windows.
The latter. I was thinking about having fman "right at hand" anywhere.

**Comment by mherrmann** [comment]
If you're on Windows 7, I would recommend [Launchy](https://www.launchy.net/) for launching fman from everywhere. On Win 10, wouldn't the Start menu let you launch it with a few key strokes?

**Comment by bmix** [comment]
I already use Launchy ;-) What I meant was not about launching fman, but to have it hidden (running in the background) and activating it via a global hotkey in order to pop up to the front. Just as you would activate a terminal running Quake-style with a global hotkey. Making it hide to the notification-area would make it invisible, when not being used.

**Comment by mherrmann** [comment]
I see, and keeping it running in the background would have the advantage that showing it is superfast correct? Ie. that it circumvents fman's startup time.

**Comment by bmix** [comment]
Right! You know: "Snappy"! :-D
The presence of a programm in the taskbar is to reach it by other means, than the keyboard. Keyboard "space" is limited. But some things are so important, that they usually are always there. Terminal, filemanager, web-browser, editor, maybe email-app, instant-messanger, system-monitoring. Stuff like terminal and filemanager are superior candidates, to be hidden and to be popped into view, when needed. Oh, just like Quicksilver, when you're on Mac. You know it's always there.

**Comment by mherrmann** [comment]
I see. Can you use [RBTray](http://rbtray.sourceforge.net/) to minimise fman to the task bar, and (if necessary) [AutoHotkey](https://autohotkey.com/) to then bring it back to the foreground?

**Comment by kek91** [comment]
I think the notification bar should primarily be used for background programs (like an antivirus), and other notifications. Just having random software there idle kind of defeats the purpose. 

But apparantly it's becoming a trend, more and more programs defaults to the notification bar when closed and requires some magical formula to actually exit (looking at you Skype4Business). Personally I find it annoying, but I understand why they do it; to mimic the snappy feeling of mobile devices where the apps are kept open in the background (unless forcefully closed).

I don't think this should be a core feature, or even worth spending time on developing right now, but that's just my opinion :)

**Comment by mherrmann** [comment]
I second everything @kek91 said

**Comment by vovcacik** [comment]
@bmix can you rename this to "Minimize to tray" or something like that. Thx

**Comment by mherrmann** [comment]
@vovcacik I've renamed it.

**Comment by BlueInt32** [comment]
I would agree with what @kek91 said, if reopening fman didn't take 2s to launch. On windows, explorer.exe is always running in the background so it is always very fast to run. It is just not shown in the system tray. I don't know if it's possible to have a program running without being shown in the systray like explorer does, but if it is, that would be a nice improvement for fman.

**Comment by cja-github** [comment]
"I think the notification bar should primarily be used for background programs (like an antivirus), and other notifications. Just having random software there idle kind of defeats the purpose."
- Program design should be pragmatic. fman is an Explorer alternative, and Explorer "cheats" by running all the time with no systray icon. For me it's more important that fman starts immediately when I need it than the number of icons in my systray. It can always be configurable for the people who prefer to wait in order to have fewer systray icons. But for me, one more icon here really isn't a problem:
![image](https://user-images.githubusercontent.com/5276879/181471065-af13a23e-d3be-4ccd-9a72-cf796dbc0a88.png)
